### PR TITLE
Switch to mda theme

### PR DESCRIPTION
**Pull request by lilyminium**
https://github.com/MDAnalysis/mdanalysis-sphinx-theme/issues/33

----
:books: Documentation preview :books:: https://mdanalysis--307.org.readthedocs.build/en/307/

### CODE REVIEW

**Review by IAlibay — COMMENTED** [review]
Should this still be there? https://github.com/MDAnalysis/UserGuide/pull/307/files#diff-d8d3ed25802824d15bf411f8e97416d8fc6f9247e821b0617f2b869dc584b99cR82

**Comment by lilyminium** [comment]
No you're right, thanks!

File: doc/source/conf.py [hunk 1]
```diff
@@ -14,12 +14,8 @@
...
 
 import datetime
-import subprocess
-from collections import OrderedDict
-
 import MDAnalysis as mda
-import msmb_theme
-import sphinx_rtd_theme
+import subprocess
 from ipywidgets.embed import DEFAULT_EMBED_REQUIREJS_URL
```
Comment (IAlibay) [inline doc/source/conf.py]: Whilst we're here, are we using this anywhere anymore?

File: doc/source/conf.py [hunk 2]
```diff
@@ -155,33 +132,19 @@ def sort_authors(filename):
...
-# pixels large.
-html_favicon = "_static/logos/mdanalysis-logo.ico"
 html_logo = "_static/logos/user_guide.png"
 
 html_context = {"versions_json_url": "https://userguide.mdanalysis.org/versions.json"}
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
```
Comment (IAlibay) [inline doc/source/conf.py:183]: We still have stuff in `_static` (thinking of the above `user_guide.png`, should we keep this around?

**Review by IAlibay — APPROVED** [review]
Two small questions, otherwise lgtm!

**Review by IAlibay — APPROVED** [review]
lgtm, thanks @lilyminium !

File: doc/source/conf.py [hunk 3]
```diff
@@ -14,13 +14,8 @@
 # import sys
 
 import datetime
-import subprocess
-from collections import OrderedDict
-
 import MDAnalysis as mda
```
Comment (IAlibay) [inline doc/source/conf.py]: There's some pre-commit failure with import orders and an extra blank line on 124 if that matters to you.

File: doc/source/conf.py [hunk 4]
```diff
@@ -15,12 +15,7 @@
 
 import datetime
 import subprocess
-from collections import OrderedDict
-
 import MDAnalysis as mda
```
Comment (IAlibay) [inline doc/source/conf.py:19]:
```suggestion

import MDAnalysis as mda
```
🤣 sorry I can't help myself from this

**Comment by IAlibay** [comment]
That is a looonnnng RTD build (probably that specific runner but let's keep an eye out for this).

**Pull request merged**